config/browse_everything_providers.yml

```diff
@@ -0,0 +1,16 @@
+#
+# To make browse-everything aware of a provider, uncomment the info for that provider and add your API key information.
+# The file_system provider can be a path to any directory on the server where your application is running.
+#
+---
+ file_system:
```

Comment:
We have never had file_system included in the past.  I don't think we want to do that yet.  I am also not sure how the path " / " will be evaluated on a windows system.  I propose we remove it from the config.